libmamba/src/core/transaction.cpp

```diff
@@ -1562,6 +1580,60 @@ namespace mamba
         t.print(out);
     }
 
+    void MTransaction::printEnv()
```

Comment:
Can we make this return an object (`std::string` or even `nl::json`) instead of printing (which we're working on keeping out of libmamba)?